l1-contracts/contracts/bridge/L1AssetRouter.sol

```diff
@@ -702,32 +706,32 @@ contract L1AssetRouter is IL1AssetRouter, ReentrancyGuard, Ownable2StepUpgradeab
         bytes memory _transferData,
         bytes32 _expectedTxDataHash
     ) internal view returns (bool isLegacyTxDataHash) {
-        try this.encodeTxDataHash(true, _prevMsgSender, _assetId, _transferData) returns (bytes32 txDataHash) {
+        try this.encodeTxDataHash(0x00, _prevMsgSender, _assetId, _transferData) returns (bytes32 txDataHash) {
```

Comment:
Could you please create named constants for encoding version `0x00`, `0x01` and `0x02`?